components/locale_core/src/locale.rs

```diff
@@ -188,43 +185,6 @@ impl Locale {
         Self::normalize_utf8(input.as_bytes())
     }
 
-    /// Compare this [`Locale`] with BCP-47 bytes.
-    ///
-    /// The return value is equivalent to what would happen if you first converted this
-    /// [`Locale`] to a BCP-47 string and then performed a byte comparison.
-    ///
-    /// This function is case-sensitive and results in a *total order*, so it is appropriate for
-    /// binary search. The only argument producing [`Ordering::Equal`] is `self.to_string()`.
-    ///
-    /// # Examples
-    ///
-    /// ```
-    /// use icu::locale::Locale;
-    /// use std::cmp::Ordering;
```

Comment:
Nit: Please restore this example (using `writeable` directly) and pull it into, perhaps, the main `#Examples` section of this type